exoctk/exoctk_app/app_exoctk.py

```diff
@@ -158,9 +158,8 @@ def limb_darkening():
         form.modeldir.data = [j for i, j in form.modeldir.choices if i == form.modeldir.data][0]
 
         # Grism details
-        if '.G' in form.bandpass.data.upper() and 'GAIA' not in form.bandpass.data.upper():
-            kwargs = {'n_bins': form.n_bins.data, 'pixels_per_bin': form.n_pix.data,
-                      'wl_min': form.wave_min.data*u.um, 'wl_max': form.wave_max.data*u.um}
+        if ('.G' in form.bandpass.data.upper() and 'GAIA' not in form.bandpass.data.upper()) or form.bandpass.data.lower() == 'tophat':
+            kwargs = {'n_bins': form.n_bins.data, 'wave_min': form.wave_min.data*u.um, 'wave_max': form.wave_max.data*u.um}
```

Comment:
@hover2pi It looks like you renamed these kwargs from `wl_min` --> `wave_min`, etc. 
I don't see a change accompanying that -- was this part of the existing bug?

Reply:
It was an additional bug that was making the tophat and grism filters fail. It's the argument name that gets passed to the svo_filters.Filter class.